include/ttmlir/Target/TTNN/program.fbs

```diff
@@ -61,6 +61,12 @@ table EltwiseOp {
   out: tt.target.TensorRef;
 }
 
+table BroadcastOp {
+  in: tt.target.TensorRef;
+  out: tt.target.TensorRef;
+  dimensions: [int64];
+}
+
```

Comment:
TTNN doesn't have a broadcast op and we have the pass that bypasses it below, so we don't need this flatbuffer def.

Reply:
OK will remove it. Thanks.